### DISCUSSION TITLE
Series translation issue

### DISCUSSION THREAD

**Discussion by karihj** [open-sdg/open-sdg#1707] [Q&A] [answered]
It seems just to be us having this issue (can't see it on other Open SDG sites), but this is an annoying one :) 

![image](https://user-images.githubusercontent.com/23165808/175140160-acc2ddff-b32c-4b64-b2a0-4d4e1ee60a6f.png)
See example: https://sdg.hagstova.fo/sdg-site/1-2-1/

As you can see, the country label in the legend is translated into Faroese ("Føroyar") but the series hover label is not ("data.faroeislands"). 

In our site_config.yml we have added:
![image](https://user-images.githubusercontent.com/23165808/175141380-3895dd2b-fbd3-486d-befd-d890e37649b2.png)

We have added the translation in this file: https://github.com/hagstovan/sdg-data/blob/develop/translations/fo/data.yml 

![image](https://user-images.githubusercontent.com/23165808/175141734-ddb823cd-de3f-428c-8f65-f9a388321567.png)

Do you have any tip why it is not working?

Our site repository can be found here: https://github.com/hagstovan/sdg-site
Our data repository can be found here: https://github.com/hagstovan/sdg-data

**Comment by brockfanning** [comment, marked as answer]
@karihj I believe you've pointed out a bug in the platform, so I'll create an Issue for this.

**Reply by brockfanning** [reply]
Let's follow this here: #1708
